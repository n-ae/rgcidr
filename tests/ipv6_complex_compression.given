::1
::ffff:0:0
2001:db8::
2001:db8::1
2001:db8::8a2e:370:7334
2001:0db8:0000:0000:0000:ff00:0042:8329
2001:db8:0:0:1:0:0:1
::
2001:db8:85a3::8a2e:370:7334
2001:db8:85a3:0:0:8a2e:370:7334
fe80::
fe80::1
2001:db8:0:1:1:1:1:1
2001:0:0:1::1
0:0:0:0:0:0:0:1
0:0:0:0:0:0:0:0
2001:db8:0:0:0:0:2:1
invalid::address::double
2001:db8:::1
