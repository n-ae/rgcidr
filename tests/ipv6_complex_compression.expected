2001:db8::
2001:db8::1
2001:db8::8a2e:370:7334
2001:0db8:0000:0000:0000:ff00:0042:8329
2001:db8:0:0:1:0:0:1
2001:db8:85a3::8a2e:370:7334
2001:db8:85a3:0:0:8a2e:370:7334
2001:db8:0:1:1:1:1:1
2001:db8:0:0:0:0:2:1
